libs/contract/src/lib/bucket/+state/bucket.service.ts

```diff
@@ -108,6 +111,26 @@ export class BucketService extends CollectionService<BucketState> {
         holdbacks: contract.holdbacks,
       });
 
+      const defualtNegotiation: Negotiation = {
+        _meta: createDocumentMeta({ createdAt: new Date(), }),
+        id: contractId,
+        status: 'pending',
+        createdByOrg: orgId,
+        sellerId: centralOrgId.catalog,
+        stakeholders: [...Array.from(new Set([...parentContract.stakeholders, orgId]))],
```

Comment:
This should be enough
```suggestion
        stakeholders: [...parentContract.stakeholders, orgId],
```